### ISSUE TITLE
Released image tags should include version

### ISSUE THREAD

**Issue by bobcatfish** [tektoncd/pipeline#1232]
# Expected Behavior

If a user of Tekton wants to know what version they have installed, they should be easily able to get that information.

# Actual Behavior

To find this now you need to:
- Find the names of the images you are running, by looking at your tekton Deployments and/or finding the release.yaml you installed (e.g. release.yaml attached to https://github.com/tektoncd/pipeline/releases/tag/v0.6.0)
- Navigate in your browser (or use some kinda other tool I don't know about) to an image with its digest, e.g.  https://gcr/images/tekton-releases/GLOBAL/github.com/tektoncd/pipeline/cmd/webhook@sha256:59be4e291f510bced7c138e92cbfee1ea58e0a48969fbaa3a055c1537d77bfd9/details?tab=info) and see the tag (this one is v0.5.2)

# Additional Info

* Can we somehow include both the digest + the tag in the release.yaml? e.g. something like:
```yaml
gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:v0.6.0@sha256:43050320e994156a2683ab5d30551a52a4a59fe20b0ee50161af5d524b9aa5c6
```

I'm not sure this is actually a valid way to refer to an image tho :(

We might want to consider also including 'alpha'  / 'beta' / etc. in the tag #1231

**Comment by ImJasonH** [comment]
It is valid for an image name to include both tag and digest. In that case the digest wins but the tag can be useful informationally as in this case.

**Comment by vdemeester** [comment]
As @ImJasonH, this is possible, we need to create an issue upstream [`google/ko`](https://github.com/google/ko) to add support for having the full image reference (tag + digest) written in the resolved yaml :angel:

**Comment by vdemeester** [comment]
This has been fixed since 0.10 at least, closing.

/close
